dts/bindings/ipc/nordic,nrf-ipct-global.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic Global IPCT (Interprocessor Communication Transceiver)
+
+compatible: "nordic,nrf-ipct-global"
+
+include: ["nordic,nrf-ipct-common.yaml", "nordic,split-channels.yaml"]
+
+properties:
+  global-domain-id:
+    required: true
+    type: int
+    description: |
+      Global IPCT instances reside on specific buses within the Global Domain,
+      such as fast and slow, which have different IDs that do not match the
+      standard Global Domain ID presented in their address.
```

Comment:
question: what is the real purpose of this property? Feels like the `domain` one that existed for mbox-nrf-ids

Reply:
I'm not sure. I saw that it is used by regtool so I thought it should be kept. @57300 Could you comment on that?

Reply:
can domain be inferred by other means, like it happened with ipc related stuff? I'd like to make sure we do not add redundancy in DT just for a tool.

Reply:
What is redundant here? This can't be inferred from the address, as the description says, because our brilliant hardware designers assigned both IPCT instances to domain ID 15, instead of 12 and 13.

The IPC stuff we solved by parsing node labels, which I still don't think is great. Do you want to do the same here, and label the nodes `globalfast_ipct` and `globalslow_ipct`?

Reply:
I see your point. To me, both options are equally bad (nodelabels adding info, or, adding properties not consumed by driver code). Sad that HW has been laid out like this. Let's take the shortest path here, we can always address this later.